app/lib/widgets/CommentView.dart

```diff
@@ -22,162 +23,148 @@ class CommentView extends StatefulWidget {
-                onPressed: (BuildContext context) {},
-                backgroundColor: AppCommonTheme.backgroundColor,
-                foregroundColor: Colors.white,
-                icon: Icons.report,
-              ),
-            ],
-          ),
-          child: Flex(
-            direction: Axis.horizontal,
-            children: [
-              const CircleAvatar(backgroundColor: Colors.white),
-              Expanded(
-                // fit: FlexFit.loose,
-                child: Padding(
-                  padding: const EdgeInsets.only(left: 12.0),
-                  child: Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: [
-                      Text(
-                        widget.name,
-                        style: TextStyle(
-                          color: widget.titleColor,
-                          fontSize: 16,
+    return GestureDetector(
+      onDoubleTap: () {
+        newsCommentController.handleCommentLikeClick(widget.postition);
+      },
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Slidable(
+            endActionPane: ActionPane(
+              motion: const ScrollMotion(),
+              children: [
+                SlidableAction(
+                  onPressed: (BuildContext context) {},
+                  backgroundColor: AppCommonTheme.backgroundColor,
+                  foregroundColor: Colors.white,
+                  icon: Icons.reply,
+                ),
+                SlidableAction(
+                  onPressed: (BuildContext context) {},
+                  backgroundColor: AppCommonTheme.backgroundColor,
+                  foregroundColor: Colors.white,
+                  icon: Icons.report,
+                ),
+              ],
+            ),
+            child: Flex(
+              direction: Axis.horizontal,
+              children: [
+                const CircleAvatar(backgroundColor: Colors.white),
+                Expanded(
+                  // fit: FlexFit.loose,
+                  child: Padding(
+                    padding: const EdgeInsets.only(left: 12.0),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(
+                          widget.commentModel.name,
+                          style: TextStyle(
+                            color: widget.commentModel.titleColor,
+                            fontSize: 16,
+                          ),
                         ),
-                      ),
-                      Text(
-                        widget.comment,
-                        style: const TextStyle(
-                          color: Colors.white,
-                          fontSize: 16,
+                        Text(
+                          widget.commentModel.comment,
+                          style: const TextStyle(
```

Comment:
Please add theme constants in ```seperatedThemes.dart``` if reusable.

Reply:
we cannot as right now I am passing color from the previous screen.. once API is integrated... we will set colors randomly